coala_quickstart/info_extraction/InfoExtractor.py

```diff
@@ -14,7 +16,16 @@ def __init__(self,
         :param project_directory: Absolute path to project directory in which
                                   the target files will be searched.
         """
-        self.target_globs = target_globs
+        target_files = self.retrieve_files(target_globs, project_directory)
+        for fname in target_files:
+            if not fnmatch(fname, self.supported_file_globs):
+                raise ValueError("The taraget file {} does not match the "
```

Comment:
will this ever happen? Doesn't `glob` only return files that actually match the glob?